Fix the comments w.r.t. configuration file usage with the -T option.

The changes have been submitted upstream.

--- snort.8.orig	2014-09-25 07:44:55.175565999 -0700
+++ snort.8	2014-09-26 11:19:43.998692220 -0700
@@ -339,8 +339,7 @@
 indicating that everything is ready to proceed.  This is a good
 switch to use if daemon mode is going to be used, it verifies that
 the Snort configuration that is about to be used is valid and won't fail at
-run time. Note, Snort looks for either /etc/snort.conf or ./snort.conf.
-If your config lives elsewhere, use the -c option to specify a valid
+run time. Note that you will need to use the -c option to specify a valid
 .I config-file.
 .IP "-u user"
 Change the user/UID Snort runs under to
